app/views/comfy/blog/posts/show.html.haml

```diff
@@ -1,5 +1,12 @@
 %h1= @cms_post.title
 
+- if @cms_post.file.present? && @cms_post.file.attachment.attached?
+  - attachment = @cms_post.file.attachment
+  = link_to url_for(attachment) do
+    - if attachment.content_type.include? "image"
```

Comment:
Ideally, this would be a model validation or method but I didn't want to take any liberties regarding the base GEM.